ee_tests/src/booster_ee_int_tests/booster_pipeline.spec.ts

```diff
@@ -29,9 +30,83 @@ describe('Verify the completion of the build pipeline:', () => {
     await dashboardPage.logout();
   });
 
-  xit('Login, test, logout', async () => {
-    // TODO: implement
-    info('Not yet implemented');
+  it('Login, test deployment to stage and run, logout', async () => {
+    let spaceName = support.currentSpaceName();
+    let spaceDashboardPage = new SpaceDashboardPage(spaceName);
+    await spaceDashboardPage.openInBrowser();
+    await spaceDashboardPage.pipelinesSectionTitle.clickWhenReady(support.LONGER_WAIT);
+    support.debug('Accessed pipeline page');
+
+    let spacePipelinePage = new SpacePipelinePage();
+    globalSpacePipelinePage = spacePipelinePage;
+
+    let pipelineByName = new Button(spacePipelinePage.pipelineByName(spaceName), 'Pipeline By Name');
+
+    support.debug('Looking for the pipeline name');
+    await pipelineByName.untilPresent(support.LONGER_WAIT);
+
+    /* Verify that only (1) new matching pipeline is found */
+    support.debug('Verifying that only 1 pipeline is found with a matching name');
+    expect(await spacePipelinePage.allPipelineByName(spaceName).count()).toBe(1);
+
+    /* Find the pipeline name */
+    await pipelineByName.untilClickable(support.LONGER_WAIT);
+
+    /* Verify stage and run icons are present - these will timeout and cause failures if missing */
+    await spacePipelinePage.stageIcon.untilClickable(support.LONGEST_WAIT);
+    await spacePipelinePage.runIcon.untilClickable(support.LONGEST_WAIT);
```

Comment:
@sunilk747 I ran the test locally but it did not finish. The `runIcon` is displayed only after the build is approved:
 * Click on `Input Required` button
 * Click on `Approve` button

which is missing here. So the test gets stuck and it's time runs out.

Please add the code for approving the build. Thanks

Reply:
ok. Will do it.